compiler/ilgen/IlBuilder.cpp

```diff
@@ -1212,6 +1217,30 @@ IlBuilder::AddWithOverflow(TR::IlBuilder **handler, TR::IlValue *left, TR::IlVal
    }
 
 TR::IlValue *
+IlBuilder::UnsignedAddWithOverflow(TR::IlBuilder **handler, TR::IlValue *left, TR::IlValue *right)
```

Comment:
Thanks for all the suggestions. @mstoodle Could you please take a look? Any suggestions will be appreciate. :)